### PR TITLE
Add index.html file and linters configuration

### PR DESCRIPTION
**Pull request by Artufelii**
Create of index.html file with initial code, and alse add the linters configuration necessary for the tests on HTML and CSS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+
+- [👥 Authors](#authors)
```
Comment (didierganthier) [inline README.md]: Well done on using the README template provided by Microverse 💯 but several sections seem to have been removed from the `Table of Contents` 📄 these sections were required, only the `FAQ` is optional and can be removed. Make sure to add the other sections to make the README complete just like the [template](https://github.com/microverseinc/readme-template/blob/master/README.md) 🤓

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "18.x"
```
Comment (didierganthier) [inline .github/workflows/linters.yml]: Nice work on the linters ✅ but in order to fix the failing lighthouse ❌ you could try replacing that line by this:
```
node-version: "12.x"
```
The lighthouse might get fixed then 😉

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @Artufelii ,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR ✔️ 
- Added `.gitignore` ✔️ 
- Used README template ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Comment by Artufelii** [comment]
Ok thank your for your help and I will do the changes

**Comment by Artufelii** [comment]
I already do the changes on the README file and linters file, thanks for your help @didierganthier

File: README.md [hunk 3]
```diff
@@ -1 +1,205 @@
...
+
+# 📖 [Microverse Hello World] <a name="about-project"></a>
+
+Microverse hello world is an app with the porpose of develop the basic skills of html and css
+
+## 🛠 Built With HTML and CSS <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] You did well adding the other sections as suggested by the previous reviewer. however, kindly always delete the part with the `>` sign, as it's added to show you directions o what t add. This helps keep your code clean, and adhere to professionalism 👍

File: README.md [hunk 4]
```diff
@@ -1 +1,205 @@
...
+  <ul>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Please revert to my detailed comment on line `42` on the comment here 👍

File: README.md [hunk 5]
```diff
@@ -1 +1,205 @@
...
+
+- it's a basic web page using HTML and CSS
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Please revert to my detailed comment on line `42` on the comment here 👍

File: README.md [hunk 6]
```diff
@@ -1 +1,205 @@
...
+
+- [Live Demo Link](https://artufelii.github.io/microverse-hello-world/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Please revert to my detailed comment on line `42` on the comment here 👍

File: README.md [hunk 7]
```diff
@@ -1 +1,205 @@
...
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Please revert to my detailed comment on line `42` on the comment here 👍

File: README.md [hunk 8]
```diff
@@ -1 +1,205 @@
...
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Please revert to my detailed comment on line `42` on the comment here 👍

File: README.md [hunk 9]
```diff
@@ -1 +1,205 @@
...
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Please revert to my detailed comment on line `42` on the comment here 👍

File: README.md [hunk 10]
```diff
@@ -1 +1,205 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Nice one adding the MIT license 👏 . However, you need to have a file named `LICENSE` at the root of your project, with the content given on the README template, so the path here is not broken. 
-  [ ] Please do delete line `203`, as its meant to serve as a guide and not to be commented on your code 👍 . This keeps your code clean, and give a more professional feel

File: README.md [hunk 11]
```diff
@@ -1 +1,205 @@
...
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] The `FAQ` is the only optional section as said by your previous reviewer. if you don't intend to add to it, kindly delete the whole section. Else, I strongly advice you to add a possible question people might ask about your project

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Artufelii      👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 



Highlights
✅All linters are checked

  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required below need to be implemented :
- You had a lot of commented guides from the template from the new addition you made. As guides, they should not be added to your project. Please check for a detailed review below 👍 


_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 11 above)
Comment (Artufelii) [reply README.md]: thank you @Shedrack-Sunday I did the changes already

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @Artufelii, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**